recipes/impyla/meta.yaml

```diff
@@ -0,0 +1,42 @@
+package:
+  name: impyla
+  version: "0.13.6"
+
+source:
+  git_url: https://github.com/cloudera/impyla
+  git_rev: v0.13.6
+
+build:
+  number: 0
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - six
+    - bitarray
+    - thrift # [py2k]
```

Comment:
Is this a mandatory dependency? If so `impyla` is not `py3k` ready and you need yo skip the build like you did [here](https://github.com/conda-forge/staged-recipes/pull/315/files#diff-7a93cc588f219554bfe98f859ece75e1R12).